Equipment.md

```diff
@@ -59,28 +61,28 @@ You can then select any two of the following properties to further customize you
 * __[Reach]__: On melee weapons greater than 5 feet in length, increase the range to 10 feet. On Ranged weapons, double the range.
 
 #### Dice Ranks
-|Dice|Average|||Dice|Average|
-|----|-------|---|---|---|---|
-|3d12| (19.5)|||2d6 |  (7)  ||
-|3d10| (16.5)|||1d12| (6.5) ||
-|3d8 | (13.5)|||1d10| (5.5) ||
-|2d12| (13)  |||2d4 |  (5)  ||
-|2d10| (11)  |||1d8 | (4.5) ||
-|3d6 | (10.5)|||1d6 | (3.5) ||
-|2d8 |  (9)  |||1d4 | (2.5) ||
-|3d4 | (7.5) ||
-
+|Dice|Average|
+|----|----|
+|1d6 | 3.5|
+|1d8 | 4.5|
+|1d10| 5.5|
+|2d6 |   7|
+|2d8 |   9|
+|2d10|  11|
+|3d8 |13.5|
+|3d10|16.5|
+|3d12|19.5|
 
 ### Example Weapons
-* __Greataxe__ is a [Brutal 2] Huge Slashing weapon. Its damage is 2d12 + Strength + your level, critical 19-20 x2, range 5 feet.
-* __Lance__ is a [Devastating] [Reach] Huge Piercing weapon. Its damage is 3d6 + Strength, critical 18-20 x5, range 10 feet.
-* __Longbow__ is a [Brutal 2] Huge Ranged weapon. Its damage is 2d12 + Strength + level, critical 20 x4, range 300 feet.
+* __Greataxe__ is a [Brutal 2] [Devastating] Huge Slashing weapon. Its damage is 3d8 + Strength + your level, critical 19-20 x3, range 5 feet.
+* __Lance__ is a [Devastating] [Reach] [Heavy] Huge Piercing weapon. Its damage is 2d10 + Strength, critical 18-20 x5, -2 on attack rolls, range 10 feet.
+* __Longbow__ is a [Brutal 2] [Reach] Huge Ranged weapon. Its damage is 3d8 + Strength + level, critical 20 x4, range 600 feet.
 * __Longsword__ is a [Brutal] [Parrying] Long Slashing weapon. With it, your AC is 2 higher. Its damage is 2d8 + Strength + one-half your level, critical 19-20 x2, range 5 feet.
-* __Quarterstaff__ is a [Deft] [Parrying] Long Bludgeoning weapon, requiring 2 hands. With it, your initiative and AC are 2 higher. It has a +1 to attack rolls, damage 2d8 + Strength, critical 20 x3, range 5 feet.
-* __Rapier__ is a [Heartseeker] [Parrying] Medium Piercing weapon. With it, your AC is 2 higher. Its damage is 1d6 + Strength, critical 17-20 x4, range 5 feet.
-* __Shiv__ is a [Deft] [Devastating] Short Piercing weapon. With it, your initiative is 2 higher. It has a +1 to attack rolls, damage 1d4 + Strength, critical 18-20 x5, range 5 feet.
-* __Shortbow__ is a [Brutal] [Heartseeker] Medium Ranged weapon. Its damage is 1d8 + Strength, critical 19-20 x3, range 60 feet.
-* __Sling__ is a [Heavy] [Brutal] Short Ranged weapon. Its damage is 1d8 + Strength + one-half level, critical 19-20 x3, range 30 feet.
+* __Quarterstaff__ is a [Deft] [Parrying] Long Bludgeoning weapon, requiring 2 hands. With it, your initiative and AC are 2 higher. It has a +1 to attack rolls, damage 2d10 + Strength, critical 20 x3, range 5 feet.
+* __Rapier__ is a [Heartseeker] [Parrying] Medium Piercing weapon. With it, your AC is 2 higher. Its damage is 1d10 + Strength, critical 17-20 x4, range 5 feet.
+* __Shiv__ is a [Deft] Short Piercing weapon. With it, your initiative is 2 higher. It has a +1 to attack rolls, damage 1d6 + Strength, critical 19-20 x5, range 5 feet.
```

Comment:
Would rather Shiv stays [Devastating] than [Deft], since it's basically CritFishing.txt.

Reply:
Heard, fixed.